src/java.base/share/classes/java/lang/invoke/MethodHandleImpl.java

```diff
@@ -1552,8 +1553,8 @@ public Map<String, byte[]> generateHolderClasses(Stream<String> traces) {
             }
 
             @Override
-            public VarHandle memorySegmentViewHandle(Class<?> carrier, long alignmentMask, ByteOrder order) {
-                return VarHandles.memorySegmentViewHandle(carrier, alignmentMask, order);
+            public VarHandle memorySegmentViewHandle(Class<?> carrier, MemoryLayout enclosing, long alignmentMask, ByteOrder order, boolean fixedOffset, long offset) {
```

Comment:
When I was playing with the code I kept being confused by the `fixedOffset` parameter name. The reason being that no var handle is really "fixed offset" (all VH take a "base" offset -- and this doesn't change that). What this "fixedOffset" means is really "there's no other (dynamic) offset on top of the base offset". I think calling it "stridedAccess" seems subjectively more evocative (or something like that).

Reply:
Shouldn't this be called noStride then?

Reply:
noStride works (I was mostly suggesting that appealing to the concept of "there's a stride or not" might be more useful)